### DISCUSSION TITLE
[Question] Disable sway window resizing

### DISCUSSION THREAD

**Discussion by Misterio77** [Alexays/Waybar#1823] [Q&A]
Hello! Thanks to all developers and maintainers for waybar, i switched from xorg to wayland and am loving the experience.

I'm not sure if this has been asked before, but is there any configuration for disabling the behaviour that resizes sway windows when waybar is launched?

I _think_ it may happen on waybar side. ~I suspect it happens in [this code](https://github.com/Alexays/Waybar/blob/bcb63b8ccb058110c3c3ce280ea653917a9261cc/src/modules/sway/ipc/client.cpp)~ Nope, that's just the sway module.

I know this is kinda specific to my workflow, but i wanted to disable this behaviour for two reasons:
- Toggling the bar without resizing the entire desktop
- Reloading the bar (i use [flavours](https://github.com/misterio77/flavours) to quickly change desktop color schemes) without windows jumping around

Is this currently configurable? If not, i can try and write a PR for it, if the maintainers think it fits the project and would be useful.

Thanks <3

**Comment by Alexays** [comment]
Hi, check the layer option from config file :)

**Comment by Misterio77** [comment]
Hey, that behaviour seems to occur with `layer` set to either `bottom` or `top`:
![behaviour](https://u.cubeupload.com/Misterio77x/waybar.gif)

**Comment by Misterio77** [comment]
Sorry for bothering, but i'm still facing this issue. Waybar still makes windows jump when restarted or reloaded, this occurs with both `bottom` and `top`.

**Comment by notginned** [comment]
+1, having that issue on the latest v.0.9.7 too, despite whichever layer option I choose
